src/navigation/Tabs.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react"
+
+import "./Tabs.scss"
+
+import {
+  Tab as ReactTab,
+  Tabs as ReactTabs,
+  TabList as ReactTabList,
+  TabPanel as ReactTabPanel,
+} from "react-tabs"
+
+export interface TabsProps {
+  children: React.ReactNode
+  className?: string
+  defaultFocus?: boolean
+  defaultIndex?: number
+  disabledTabClassName?: string
+  domRef?: (node?: HTMLElement) => void
+  focusTabOnClick?: boolean
+  forceRenderTabPanel?: boolean
+  onSelect?: (index: number, last: number, event: Event) => boolean | void
+  selectedIndex?: number
+  selectedTabClassName?: string
+  selectedTabPanelClassName?: string
+}
+
+const Tabs = (props: TabsProps) => {
+  const className = ["tabs"]
+  if (props.className) className.push(props.className)
+  const focusTab = typeof props.focusTabOnClick !== "undefined" ? props.focusTabOnClick : false
```

Comment:
This is more of a curiosity. Why do you need to do "typeof" to see if it is undefined? Couldn't you just do this?
```
const focusTab = props.focusTabOnClick !== undefined ? props.focusTabOnclick : false
```

Reply:
Ah, just force of habit — if a variable name was never declared initially, it won't throw an error to use `typeof`: https://stackoverflow.com/a/16465346/551775

But here we're checking a known object's property, so even on a missing value it wouldn't ever throw an error to use your recommended syntax.